src/@types/document.ts

```diff
@@ -44,8 +44,7 @@ export interface WrappedDocument<T extends OpenAttestationDocument = OpenAttesta
 }
```

Comment:
can you check that everything that has been added in the types is really used ? https://github.com/Open-Attestation/open-attestation/pull/92/files?file-filters%5B%5D=.ts#diff-2147217712d9b535abfe6cc9568e6661R15

Reply:
yep, I've checked against the w3c document, its quite complete. In my implementation in oa-verify I'll have to add the `created` field since I've missed out on that.